Sources/Private/Model/DotLottie/DotLottieAnimationMode.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  DotLottieAnimationMode.swift
+//  Lottie
+//
+//  Created by Evandro Hoffmann on 09/06/23.
+//
+
+import Foundation
+
+enum DotLottieAnimationMode: String, Codable {
```

Comment:
Can we just put this in `DotLottieAnimation.swift` instead of a putting it in a new file?